Tasks/DownloadFileShareArtifactsV0/task.json

```diff
@@ -0,0 +1,80 @@
+{
+  "id": "E3CF3806-AD30-4EC4-8F1E-8ECD98771AA0",
+  "name": "DownloadFileShareArtifacts",
+  "friendlyName": "Download FileShare Artifacts",
+  "description": "Download FileShare Artifacts",
+  "helpMarkDown": "",
+  "category": "Utility",
+  "author": "Microsoft Corporation",
+  "version": {
+    "Major": 0,
+    "Minor": 136,
+    "Patch": 0
+  },
+  "groups": [
+    {
+        "name": "advanced",
+        "displayName": "Advanced",
+        "isExpanded": false
```

Comment:
I prefer having all groups expanded by default or not using groups at all unless needed. The way groups show up in the UI, most customers don't expand them, so discoverability is less.

Reply:
change to be expanded by default.